specs/phase1/beacon-chain.md

```diff
@@ -401,6 +418,17 @@ class ShardTransition(Container):
     proposer_signature_aggregate: BLSSignature
 ```
 
+### `ShardTransitionCandidate`
+
+```python
+class ShardTransitionCandidate(Container):
+    transition_root: Root
```

Comment:
Should these be ordered similar to `ShardTransition`? (eg. uint64 before byte32)